src/main/webapp/listings.html

```diff
@@ -6,11 +6,11 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body onload="loadListings();">
+  <body onload="loadListings(); fetch_auth_info();">
```

Comment:
I believe `loadListings()` already calls `fetch_auth_info()`, so I think you can take this second call out

Reply:
got it i'll take out the call to fetch auth info sine its in load listings